src/components/StatusBar/index.tsx

```diff
@@ -39,18 +40,24 @@ const StatusBar = () => {
             <Typography className={"status-message"}>
                 {/* This is left blank intentionally until status messages are implemented. */}
             </Typography>
-            <Button
-                color={"primary"}
-                disabled={isDisabled(uiState, UI_ELEMENT.LOG_EVENT_NUM_DISPLAY)}
-                size={"sm"}
-                variant={"soft"}
-                onClick={handleCopyLinkButtonClick}
-            >
-                {"Log Event "}
-                {logEventNum}
-                {" / "}
-                {numEvents}
-            </Button>
+
+            <Tooltip title={"Copy link to clipboard"}>
+                <span>
+                    <Button
+                        color={"primary"}
+                        disabled={isDisabled(uiState, UI_ELEMENT.LOG_EVENT_NUM_DISPLAY)}
+                        size={"sm"}
+                        variant={"soft"}
+                        onClick={handleCopyLinkButtonClick}
+                    >
+                        {"Log Event "}
+                        {logEventNum}
+                        {" / "}
+                        {numEvents}
+                    </Button>
+                </span>
+            </Tooltip>
+
             <LogLevelSelect/>
```

Comment:
```suggestion
            <Tooltip title={"Change log level"}>
                <span>
                    <LogLevelSelect/>
                </span>
            </Tooltip>
```
Should we also include a tooltip for log level filter button?

Reply:
That was intentionally left out as it was previously discussed in #77 that the purpose of the dropdown was obvious. That said, feel free to raise your arguments.

Reply:
yeah if that is intended then np :)